TempleEight/docs/guide/orchestration.md

```diff
@@ -132,6 +132,19 @@ The `push-image.sh` script builds Docker images from each service generated in y
 Since Kubernetes requires all images used in your system to be hosted in a [Docker Registry](https://docs.docker.com/registry/), Temple's Kubernetes infrastructure features its own registry, hosted in the cluster itself.
 This script pushes all of the built images to this registry for K8s to use.
 
+:::info
+Due to a [known issue in Docker for Mac](https://github.com/docker/for-mac/issues/3611), pushing to a local registry doesn't work by default.
+Until this is fixed, a simple work around is to change the following line in your `/etc/hosts` file:
+```
+127.0.0.1	localhost
+```
+Change it to:
+```
+127.0.0.1	localhost registry.me
+```
+Then set the `$REG_URL` environment variable to be `registry.me` before running the `deploy.sh` script.
```

Comment:
Ahh - I don't think that's currently how it's generated, but I think we should make it so it works like this.
ATM we generate:

```bash
#!/bin/sh
REGISTRY_URL="localhost:5000"
BASEDIR=$(dirname "$BASH_SOURCE")

for service in "seller" "customer" "product-category" "product" "amazon-order" "auth"; do
  docker build -t "$REGISTRY_URL/amazon-$service" "$BASEDIR/$service"
 docker push "$REGISTRY_URL/amazon-$service"
done
```

But if we rename to $REG_URL, then check if it's already set, otherwise default to localhost?

Reply:
Ahhh yes true, in the kube deploy it still checks $REG_URL I believe